test/performance/rtdStorage.js

```diff
@@ -0,0 +1,84 @@
+import { group, sleep } from 'k6'
+import { putBlob } from '../common/api/rtdStorage.js'
+import { assert, statusCreated } from '../common/assertions.js'
+import { isEnvValid, isTestEnabledOnEnv, DEV, UAT } from '../common/envs.js'
+import dotenv from 'k6/x/dotenv'
+import exec from 'k6/execution';
+
+const REGISTERED_ENVS = [DEV, UAT]
+
+const services = JSON.parse(open('../../services/environments.json'))
+const BLOB_PREFIX = 'CSTAR.K6000.TRNLOG.'
+const BLOB_SUFFIX = '.NNN.csv.pgp'
+export let options = {
+	vus: 9,
+	duration: '1m',
+  }
+let params = {}
+let baseUrl
+let myEnv
+let payload
+let n = 1
+
+if (isEnvValid(__ENV.TARGET_ENV)) {
+	myEnv = dotenv.parse(open(`../../.env.${__ENV.TARGET_ENV}.local`))
+	baseUrl = services[`${__ENV.TARGET_ENV}_issuer`].baseUrl
+
+	options.tlsAuth = [
+		{
+			domains: [baseUrl],
+			cert: open(`../../certs/${myEnv.MAUTH_CERT_NAME}`),
+			key: open(`../../certs/${myEnv.MAUTH_PRIVATE_KEY_NAME}`),
+		},
+	]
+
+	params.headers = {
+		'Ocp-Apim-Subscription-Key': myEnv.APIM_RTDPRODUCT_SK,
+		'Ocp-Apim-Trace': 'true',
```

Comment:
Maybe a performance test doesn't need to be traced. It could be just impossible due to the number of requests

Reply:
Nice catch, I'll fix it